server/config.ts

```diff
@@ -86,5 +86,6 @@ export default {
     tagManagerContainerId: get('TAG_MANAGER_CONTAINER_ID', ''),
     tagManagerEnvironment: get('TAG_MANAGER_ENVIRONMENT', ''),
   },
-  enabledPrisons: process.env.ENABLED_PRISONS,
+  enabledPrisons: get('ENABLED_PRISONS', '', requiredInProduction).split(','),
+  confirmEnabled: get('CONFIRM_ENABLED', 'false', requiredInProduction) === 'true',
```

Comment:
using helper method that ensures set in production! also moving parsing of data to boundary of the system rather than doing the split of caseloads on every call